simpleflow/swf/process/actor.py

```diff
@@ -214,10 +214,10 @@ def start(self):
         self.set_process_name()
         while self.is_alive:
             try:
-                task = self._poll(self.task_list, self.identity)
+                response = self._poll(self.task_list, self.identity)
```

Comment:
Or course not :/ And unfortunately it's not the only incorrect doc. Do you have a tool or something to check this? Integrated in Pycharm somehow?